bedrock/settings/__init__.py

```diff
@@ -79,6 +79,17 @@
     }
 }
 
+# cache for generated QR codes
+CACHES['qrcode'] = {
+    'BACKEND': 'bedrock.base.cache.SimpleDictCache',
+    'LOCATION': 'qrcode',
+    'TIMEOUT': None,
+    'OPTIONS': {
+        'MAX_ENTRIES': 20,
+        'CULL_FREQUENCY': 4,  # 1/4 entries deleted if max reached
```

Comment:
Just so I understand, if we have more than 20 generated QRCodes on the site at one time and a full cache, what would be the effect on performance?

Reply:
it would mean a few more times generating the svg. The idea is that if it grows too large it will delete a quarter of the entries in the cache, which will then be generated again if they're requested. We can always grow the size of the cache if we see issues and the size isn't negatively impacting resources.